PublicMapping/districtbuilder
# Ignore unassigned districts when computing flag for Contiguity and Equal Population(*)

## Overview

- Ignores districts without assigned areas when computing contiguity and equal population flags in evaluate mode

### Checklist


### Demo

![image](https://user-images.githubusercontent.com/66973361/119887601-56adc280-bf02-11eb-84bf-40fd5269bd6f.png)


### Notes

* I added similar logic for the Equal Population metric, since it doesn't really make sense that it would fail if all of the districts created were acceptable. However, thinking through this further - does it make sense to add a 3rd 'incomplete' state for these flags? I think this might help to acknowledge that the plan is not truly valid yet, since there are still unassigned areas and missing districts.

## Testing Instructions

1. Create a map
2. Add one district that is contiguous and has a population within the accepted deviation tolerance
3. Go to evaluate mode and review metrics - expect that the contiguity and equal population flags are both displayed as true

Closes #749
